dev-manual/api/api-reference.rst

```diff
@@ -9,20 +9,138 @@ Quickly review all available resources for the Archivematica API and Storage Ser
+
+Start Transfer
+^^^^^^^^^^^^^^
+
+========  ==================================  =================================
+``POST``  **/api/transfer/start_transfer/**   *Initiates a transfer in Archivematica.*
+========  ==================================  =================================
+
+Request body parameters:
+
+.. tip:: The parameters in the following table can be submitted as a JSON
+   object with key-value pairs in the request body.
+
+=============  ================================================================
+
+``name``           Name of transfer
+
+``type``           Type of the new transfer (e.g. standard, unzipped bag,
+                   zipped bag, dspace).
+
+``accession``      Accession number of new transfer.
+
+``paths[]``        List of base64-encoded
+                   ``<location_uuid>:<relative_path>`` to be copied into the new transfer. Location UUIDs should be associated with this pipeline, and relative path should be relative to the location.
+
+``row_ids[]``      ID of the associated TransferMetadataSet for disk image
+                   ingest. Can be provided as ``[""]``.
+
+=============  ================================================================
+
+Example request:
+
+.. literalinclude:: _code/start_transfer.curl
+
+Example response:
+
+.. literalinclude:: _code/start_transfer_response.curl
+
+.. _ingest-resource.rst:
+
+Ingest
+-------
+
+Ingest refers to the process by which digital objects are packaged into SIPs and run through several microservices, including normalization, packaging into an AIP and generation of a DIP.
+
+.. _admin-resource.rst:
+
+Administration
+--------------
+
+Administration enables you to configure various parts of the application and manage integrations and users.
+
+.. _unit-resource.rst:
+
+Unit
+----
+
+Unit refers to "a type of Package Description that is specialized to provide information about an Archival Information Unit for use by Access Aids." (OAIS, p.1-13, Section 1.7.2)
+
+.. _other-resource.rst:
+
+Other
+------
+
+Other is a generic resource category that includes path metadata associated with level of description, and it also includes name associated with any customized processing configuration.
+
+.. _beta.rst:
+
+Beta
+-----
+
+Beta refers to functionality for which the API endpoints are still in-flux and that could potentially change.
 
 Storage Service API
 ====================
 
-The Storage Service API provides access to the following categories of resources:
+The Storage Service API provides access to the following resources via endpoints:
+
+    * :ref:`ss-pipeline.rst`
+    * :ref:`ss-space.rst`
+    * :ref:`ss-location.rst`
+    * :ref:`ss-package.rst`
+
+.. _ss-pipeline.rst:
+
+Pipeline
+---------
+
+A pipeline is a representation of an Archivematica installation that is assigned a unique universal identifier (UUID) when it is registered with the Storage Service. The UUID provides information about your server and all associated clients. 
```

Comment:
```suggestion
A pipeline is a representation of an Archivematica installation that is assigned a unique universal identifier (UUID) when it is registered with the Storage Service. 
```
Depending on how accurate we need to be here, we can remove the last sentence. The UUID is really only used for identification of different pipelines. The storage service will otherwise connect to the Archivematica API to communicate with the pipeline, though not much information is sent back and forth out of the regular calls.